common/src/main/kotlin/Base58.kt

```diff
@@ -1,3 +1,6 @@
+/**
+ * A utility object for Base58 encoding and decoding.
+ */
 public object Base58Btc {
   private val ALPHABET = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
```

Comment:
What is this for?

Reply:
@nitro-neal, there's more context on the crypto branch [here](https://github.com/TBD54566975/web5-sdk-kotlin/blob/crypto/common/src/main/kotlin/web5/sdk/common/Base58Btc.kt)

Currently, Base58Btc is used for `did:key`